import/repos.json

```diff
@@ -91,7 +91,7 @@
     "branches": [
       {
         "version": "2.0.0",
-        "name": "master"
+        "name": "projection-samples"
```

Comment:
Todo once   sample PR is merged  (EventStore/EventStore-Client-NodeJS#206 ) 
 `projection-samples` ->  `master`